src/landIce/interfaceWithMPAS/Interface.cpp

```diff
@@ -714,7 +699,7 @@ void velocity_solver_extrude_3d_grid(int nLayers, int globalTrianglesStride,
 
   discretizationList->set("Workset Size", discretizationList->get("Workset Size", -1));
 
-  discretizationList->set("Method", discretizationList->get("Method", "STKExtruded")); //set to STKExtruded is not defined
+  discretizationList->set("Method", discretizationList->get("Method", "Extruded")); //set to Extruded is not defined
```

Comment:
It seems you based this PR off of a odler version of Albany, can you please rebase to this branch to Albany master? (or merge master into this branch?)

Reply:
sure, can do